src/components/interaction-designer/Connection.vue

```diff
@@ -24,12 +36,57 @@
     data() {
       return {
         // line: null, // no need to set up observers over this
+        isPermanentlyActive: false,
       }
     },
 
     computed: {
+      ...mapState('builder', ['activeConnectionContext']),
       ...mapGetters('builder', ['blocksById']),
 
+      options() {
```

Comment:
mouved out from mounted()

Reply:
@bzabos , I see what you meant. 
I made 1h research but cannot find similar `init()` possibility. For now, I would stick to current implementation. But if you find it very important we can create a new ticket to handle extra research.

I'll move this to QA for now as I consider this as minor concern.
Let me know your thoughts later when you see the comment.